### DISCUSSION TITLE
dyld: lazy symbol binding failed: Symbol not found: _g_once_impl

### DISCUSSION THREAD

**Discussion by ChildishhAlbino** [gatsbyjs/gatsby#32661] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Fatal error when trying to run as part of a Gatsby Site.

```
⠋ createSchemaCustomization
dyld: lazy symbol binding failed: Symbol not found: _g_once_impl
  Referenced from: /Users/connor.williams/Documents/emilykardumDOTcom/node_modules/gatsby-remark-images-contentful/node_modules/sharp/build/Release/sharp.node
  Expected in: /Users/connor.williams/Documents/emilykardumDOTcom/node_modules/sharp/build/Release/../../vendor/8.10.5/lib/libvips-cpp.42.dylib

dyld: Symbol not found: _g_once_impl
  Referenced from: /Users/connor.williams/Documents/emilykardumDOTcom/node_modules/gatsby-remark-images-contentful/node_modules/sharp/build/Release/sharp.node
  Expected in: /Users/connor.williams/Documents/emilykardumDOTcom/node_modules/sharp/build/Release/../../vendor/8.10.5/lib/libvips-cpp.42.dylib

[1]    61090 killed     npm start
```

### Reproduction Link

https://github.com/ChildishhAlbino/emilykardumDOTcom/tree/demo/bug-on-depdency

### Steps to Reproduce

1. git clone and npm install
2. npm start
3. wait for fail
...

### Expected Result

Site should not fail to launch

### Actual Result

Site fails to launch with error.

### Environment

```shell
System:
    OS: macOS 11.4
    CPU: (8) x64 Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.15.4 - ~/.nvm/versions/node/v14.15.4/bin/node
    npm: 6.14.10 - ~/.nvm/versions/node/v14.15.4/bin/npm
  Languages:
    Python: 3.9.1 - /usr/local/bin/python
  Browsers:
    Chrome: 92.0.4515.107
    Firefox: 85.0
    Safari: 14.1.1
  npmPackages:
    gatsby: ^2.29.1 => 2.32.13
    gatsby-image: ^2.8.0 => 2.11.0
    gatsby-plugin-advanced
```

**Comment by LekoArts** [comment]
Hi!

Something is wrong with your local installation of `sharp`, e.g. you might have installed it globally or when updating dependencies it didn't build correctly.

Checking https://github.com/lovell/sharp/issues/2337 or https://stackoverflow.com/questions/65511636/dyld-lazy-symbol-binding-failed-symbol-not-found-node-module-register-gatsby you can see that people solved it with reinstalling their setup. The MacOS instructions for node-gyp are here: https://github.com/nodejs/node-gyp#on-macos

**Reply by ChildishhAlbino** [reply]
Will try that on the weekend. 

I will ensure it's not a conflicting dependency between the `gatsby-remark-images-contentful` package and another dependency per the first recommendation in your second link but I've already tried the second response to that thread.

Thanks for the suggestions.
